tests/models/semantic_role_labeller_test.py

```diff
@@ -0,0 +1,57 @@
+import numpy
+
+from allennlp.common import Params
+from allennlp.common.tensor import arrays_to_variables
+from allennlp.data import Vocabulary
+from allennlp.data.dataset_readers import SrlReader
+from allennlp.data.fields import TextField, IndexField
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.models.semantic_role_labeller import SemanticRoleLabeller
+from allennlp.testing.test_case import AllenNlpTestCase
+
+
+class SemanticRoleLabellerTest(AllenNlpTestCase):
```

Comment:
Are we testing models save and load?  I noticed on this and the tagger, that test method isn't actually called.

And is it enough for training that `forward` works?  Presumably if `forward` doesn't crash, then backprop and parameter updates will also be fine?